### ISSUE TITLE
Proguard rules template applied to Android projects

### ISSUE THREAD

**Issue by liuzho** [jupnp/jupnp#258]
We need the proguard rules template applied to Android projects, because jupnp uses a lot of reflection, which does not work properly after proguard of Android projects.

**Comment by liuzho** [comment]
Yes, I think it would be a good idea to add ProGuard configuration instructions (provide a configuration template) in the Android section of the documentation. This is because ProGuard is almost always enabled in Android projects, and such instructions would make it more convenient to integrate jUPnP into Android projects.

I have written a ProGuard configuration for jUPnP to use in my Android project, and it works well. However, I’m not sure if there are any redundant rules (e.g., keeping classes that don’t use reflection) or whether there might be issues with LocalServer (since I haven’t used it yet). Once I finalize the optimal configuration, I will submit a PR to update the Android documentation.

**Comment by wborn** [comment]
Yes that will be useful. It could be added to the https://www.jupnp.org/docs/android page. 

Might also make sense to migrate the code examples to Kotlin on that page.

**Comment by wborn** [comment]
Is this issue fixed by https://github.com/jupnp/jupnp/pull/279?

**Comment by liuzho** [comment]
> Is this issue fixed by [#279](https://github.com/jupnp/jupnp/pull/279)?

Yes, this issue has been resolved by this commit.
Thanks to the contributors for their work and yours.
